packages/easy-email-extensions/src/AttributePanel/utils/BlockAttributeConfigurationManager.ts

```diff
@@ -1,6 +1,6 @@
 import { blocks } from '../components/blocks';
 
-type ObjectComponent = { [key: string]: () => JSX.Element | null };
+type ObjectComponent = { [key: string]: (props: any) => JSX.Element | null };
```

Comment:
Page is not the first component to have props. I'll check for better typing here.